src/core/server/logging/README.mdx

```diff
@@ -1,21 +1,31 @@
+---
+id: kibCoreLogging
+slug: /kibana-dev-docs/services/logging
+title: Logging system
+image: https://source.unsplash.com/400x175/?Logging
+summary: Core logging contains the system and service for Kibana logs.
+date: 2020-12-02
+tags: ['kibana','dev', 'contributor', 'api docs']
+---
+
 # Logging
-- [Loggers, Appenders and Layouts](#loggers-appenders-and-layouts)
-- [Logger hierarchy](#logger-hierarchy)
-- [Log level](#log-level)
-- [Layouts](#layouts)
-  - [Pattern layout](#pattern-layout)
-  - [JSON layout](#json-layout)
-- [Appenders](#appenders)
-  - [Rolling File Appender](#rolling-file-appender)
-    - [Triggering Policies](#triggering-policies)
-    - [Rolling strategies](#rolling-strategies)
-- [Configuration](#configuration)
-- [Usage](#usage)
-- [Logging config migration](#logging-config-migration)
-- [Log record format changes](#log-record-format-changes)
+- <DocLink id="kibCoreLogging" section="loggers-appenders-layouts" text="Loggers, Appenders and Layouts"/>
```

Comment:
> As long as users are viewing this from Github, it might be nice to keep the TOC in-tact.

I'd clarify with @stacey-gammon whether we strive to keep backward compatibility with Github UI. It might be hard to achieve and Docs UI can be the only officially supported UI.

Reply:
The reason I bring this up is that Github UI simply doesn't render markup it doesn't recognize, so folks looking at a README in the repo might _think_ they're seeing the complete information, when they actually aren't.

But based on my understanding of the system, BWC with Github UI isn't something we'd have control over anyway, so this may be a tradeoff we just have to live with.

Reply:
Or perhaps in new-docs-world, we keep a vanilla README.md that simply points to the URL of the live docs page, so folks don't open up `src/core/server/logging` in GH and see a bunch of partially-rendered documentation.

Reply:
@stacey-gammon Do we know yet what we want to do about supporting markdown for _both_ Github viewing and the new docs system? In other words, should we have both?

Reply:
Feedback from the team: We're going to convert docs that are only viewable via Github from `.md` to `.mdx` without changing any doc links or adding syntax that isn't supported by viewing in Github. We'll address syntax changes that are needed when the new system is closer to going live.